package.json

```
@@ -537,6 +537,13 @@
                                             "editor.action.clipboardPasteAction"
                                         ]
                                     },
+                                    {
+                                        "key": "R",
+                                        "name": "Revert the current buffer (reload from disk)",
```

```suggestion
                                        "name": "Revert the current buffer",
```
Suggest to remove the parentheses to be more concise